lib/languages.js

```diff
@@ -121,6 +121,13 @@ const languages = {
         monaco: 'asm',
         extensions: ['.asm'],
         alias: ['asm']
+    },
+    analysis: {
+        id: 'analysis',
+        name: 'Analysis tool',
+        monaco: 'asm',
+        extensions: ['.asm'],  // maybe add more? Change to a unique one?
+        alias: ['asm', 'tool']
```

Comment:
You probably want to remove the `asm` alias, as that might conflict with the `assembly` language